src/internal/operators/buffer.ts

```diff
@@ -12,7 +12,7 @@ import { OperatorFunction } from '../types';
  * <span class="informal">Collects values from the past as an array, and emits
  * that array only when another Observable emits.</span>
  *
- * ![](content/img/buffer.png)
+ * ![](/assets/images/marble-diagrams/buffer.png)
```

Comment:
This will propably work, but we append the relative path anyway, for reference see the count operator. It would be more consistent if we just use `buffer.png`

Reply:
Thanks for your feedback @JWO719 I fixed the path based on your comment.